pkg/metaclient/common.go

```diff
@@ -0,0 +1,147 @@
+package metaclient
+
+// ResponseHeader is common response header
+type ResponseHeader struct {
+	// ClusterId is the ID of the cluster which sent the response.
+	// Framework will generate uuid for every newcoming metastore
+	ClusterID string
```

Comment:
How will `ClusterID` be used in client side?

Reply:
Just a tag for different metastore backend